internal/pkg/api/server/buffer/extension/dependencies/dependencies.go

```diff
@@ -0,0 +1,97 @@
+// Package dependencies contains extension to inject dependencies to the service endpoint handlers.
```

Comment:
Nakonec jsem tuhle Goa extension pro dependencies musel duplikovat.

Je kvůli tomu, že se ty dependency interfaces pro ty dvě API liší (`buffer` nepotřebuje vše, co `templates`), takže se to sice buildovalo a spouštělo, ale padalo to u erroru stylu "dependencies is not buffer.dependencies.ForPublicRequest", když jsem udělal na to API jakýkoliv request.